DOL.WHD.Section14c.Web/src/modules/components/dolHeader/dolHeaderController.test.js

```diff
@@ -11,91 +11,8 @@ describe('formFooterControlsController', function() {
       autoSaveService
     ) {
       scope = $rootScope.$new();
-      $q = _$q_;
-
-      mockNavService = navService;
-      mockApiService = apiService;
-      mockAutoSaveService = autoSaveService;
-
-      spyOn(mockNavService, 'hasNext');
-      spyOn(mockNavService, 'hasBack');
-      spyOn(mockNavService, 'getNextSection');
-
-      saveApplication = $q.defer();
-      spyOn(mockApiService, 'saveApplication').and.returnValue(
-        saveApplication.promise
-      );
-
-      formFooterControlsController = function() {
-        return $controller('formFooterControlsController', {
-          $scope: scope,
-          navService: mockNavService
-        });
-      };
+      
     })
   );
 
-  it('save', function() {
```

Comment:
@davoodharun I'm not 100% sure on the Angular testing framework being used, but before I go dig into it, you might already have an answer: can you use the `scope` defined above to test the header's `$scope.skipToMainContent`?  I'd just like a test that verifies `$anchorScroll()` and `document.getElementById().focus()` are called.  I'm thinking something along these lines might work (with full disclosure up front that I'm not 100% clear on how the angular tests work at this point):

```javascript
var anchorScrollCalledWithID = false;
beforeEach(module(function($provide) {
  $provide.provider('$anchorScroll', function(elementID) {
    anchorScrollCalledWithID = elementID;
  });
}));

it('scrolls to the selected element', function() {
  scope.skipToMainContent('test-id');
  expect(anchorScrollCalledWithID).toBe('test-id');
});
```

etc.  I'm not trying to dictate how you implement the tests - I see what looks like Jasmine's `spyOn` in some of the other tests, so feel free to go that route if it makes more sense to you.  (I'd be surprised if my chicken scratch was anything like the optimal way to do this.  😛 )